sdk/desktopvirtualization/arm-desktopvirtualization/CHANGELOG.md

```diff
@@ -1,15 +1,70 @@
+  - Added Interface AppAttachPackageDeleteOptionalParams
+  - Added Interface AppAttachPackageGetOptionalParams
+  - Added Interface AppAttachPackageInfoImportNextOptionalParams
+  - Added Interface AppAttachPackageInfoImportOptionalParams
+  - Added Interface AppAttachPackageInfoProperties
+  - Added Interface AppAttachPackageList
+  - Added Interface AppAttachPackageListByResourceGroupNextOptionalParams
+  - Added Interface AppAttachPackageListByResourceGroupOptionalParams
+  - Added Interface AppAttachPackageListBySubscriptionNextOptionalParams
+  - Added Interface AppAttachPackageListBySubscriptionOptionalParams
+  - Added Interface AppAttachPackagePatch
+  - Added Interface AppAttachPackagePatchProperties
+  - Added Interface AppAttachPackageProperties
+  - Added Interface AppAttachPackageUpdateOptionalParams
+  - Added Interface ErrorAdditionalInfo
+  - Added Interface ErrorDetail
+  - Added Interface ErrorResponse
+  - Added Interface HostPoolsListRegistrationTokensOptionalParams
+  - Added Interface ImportPackageInfoRequest
+  - Added Interface RegistrationTokenList
+  - Added Interface RegistrationTokenMinimal
+  - Added Interface TrackedResource
+  - Added Type Alias AppAttachPackageArchitectures
+  - Added Type Alias AppAttachPackageCreateOrUpdateResponse
+  - Added Type Alias AppAttachPackageGetResponse
+  - Added Type Alias AppAttachPackageInfoImportNextResponse
+  - Added Type Alias AppAttachPackageInfoImportResponse
+  - Added Type Alias AppAttachPackageListByResourceGroupNextResponse
+  - Added Type Alias AppAttachPackageListByResourceGroupResponse
+  - Added Type Alias AppAttachPackageListBySubscriptionNextResponse
+  - Added Type Alias AppAttachPackageListBySubscriptionResponse
+  - Added Type Alias AppAttachPackageUpdateResponse
+  - Added Type Alias FailHealthCheckOnStagingFailure
+  - Added Type Alias HostPoolsListRegistrationTokensResponse
+  - Added Type Alias PackageTimestamped
+  - Added Type Alias ProvisioningState
+  - Interface ExpandMsixImage has a new optional parameter certificateExpiry
+  - Interface ExpandMsixImage has a new optional parameter certificateName
+  - Interface HostPool has a new optional parameter appAttachPackageReferences
+  - Interface PrivateEndpointConnection has a new optional parameter groupIds
+  - Interface Resource has a new optional parameter systemData
+  - Added Enum KnownAppAttachPackageArchitectures
+  - Added Enum KnownFailHealthCheckOnStagingFailure
+  - Added Enum KnownPackageTimestamped
+  - Added Enum KnownProvisioningState
+  - Interface ApplicationGroup no longer has parameter systemData
+  - Interface HostPool no longer has parameter systemData
+  - Interface PrivateEndpointConnectionWithSystemData no longer has parameter systemData
+  - Interface ResourceModelWithAllowedPropertySet no longer has parameter id
+  - Interface ResourceModelWithAllowedPropertySet no longer has parameter name
+  - Interface ResourceModelWithAllowedPropertySet no longer has parameter type
```

Comment:
is this because the properties have been moved into their parent resource?

Reply:
this model is from common-type and this change is caused by upgrade version from v1 to v5